### DISCUSSION TITLE
Question: Configurable column names

### DISCUSSION THREAD

**Discussion by IPWright83** [iddan/react-spreadsheet#347] [Q&A] [answered]
This library looks really interesting and I'm wondering if I use can use it for my use case, which is slightly different from a spreadsheet. I'm interested in your thoughts and any suggestions on an approach if you don't mind.

Basically I want a quick and easy lightweight table, to support probably up to 10k rows. First question is whether that number of rows performs ok given there's no virtualization.

I also want to be able to provide editable column names (rather than letters) and I'm wondering if there's a way to enable that, or whether hiding the letters and having a styled first row might be a better approach?

Thanks for any advice!

**Comment by iddan** [comment, marked as answer]
1. The number of rows should work okay.
2. You can provide a custom ColumnIndicator component with editing capabilities.
